README.md

```diff
@@ -1,79 +1,55 @@
 # kind-local
 
-This is a Proof of Concept of how to run a local container registry in [Kind](https://github.com/kubernetes-sigs/kind) (Kubernetes in Docker).
+This repo documents the best way to run a local [Kind](https://github.com/kubernetes-sigs/kind) cluster with Tilt.
```

Comment:
Maybe add something specific to local registry stuff: "Our recommendation is to use KinD in conjunction with a local registry for faster image pushing and pulling." or something

Reply:
done